bin/test

```diff
@@ -29,7 +29,18 @@ done
 # Verify the Practice Exercises
 for practice_exercise_dir in ./exercises/practice/*/; do
     if [ -d $practice_exercise_dir ]; then
-        echo "Checking $(basename "${practice_exercise_dir}") exercise..."
+        stub=$(echo $(basename $practice_exercise_dir) | tr - _)
+        echo "Checking ${stub} exercise..."
         # TODO: run command to verify that the example solution passes the tests
+        # create new tmp directory
+        mkdir temp
+        # copy and rename example.v file
+        cp "${practice_exercise_dir}.meta/example.v" "temp/${stub}.v"
+        # copy test v file
+        cp "${practice_exercise_dir}run_test.v" "temp/run_test.v"
+        # run tests in tmp directory
+        v run temp/run_test.v
```

Comment:
Yeah good point. There's nothing like a `--verbose` flag that we could turn on so this requires a bit more thought/effort. Because of that I think it's out of scope for this PR but it's a good suggestion I want to implement so putting it in a new issue here: https://github.com/exercism/vlang/issues/12